pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -780,6 +780,19 @@ def test_hash_equivalent(self):
         stamp = Timestamp(datetime(2011, 1, 1))
         assert d[stamp] == 5
 
+    def test_tz_conversion_freq(self, tz_naive_fixture):
+        # GH25241
+        t1 = Timestamp('2019-01-01 10:00', freq='H')
+        assert t1.tz_localize(tz=tz_naive_fixture).freq == t1.freq
+        t2 = Timestamp('2019-01-02 12:00', tz='UTC', freq='T')
+        assert t2.tz_convert(tz='UTC').freq == t2.freq
+
+        # freq propagation error doesn't happen with DatetimeIndex
+        t3 = DatetimeIndex(['2019-01-01 10:00'], freq='H')
```

Comment:
These are better suited in `pandas/tests/indexes/datetimes/test_timezones.py`. After moved, LGTM

Reply:
moved! thanks! @mroeschke